### ISSUE TITLE
Prevent page turn & animation?

### ISSUE THREAD

**Issue by lukemorcom** [Nodlik/StPageFlip#25]
Hi,

I am using this library to create an interactive book that contains videos/links on the page. When clicking the controls in the video, or links on the page, it causes the page to turn. Is there any way I can prevent this event from triggering, when clicking a certain component with a specific className?

**Comment by eplicanic** [comment]
Yes. Making turn events on the pages would make this plugin much more useful.

**Comment by emir-plicanic-14243** [comment]
It looks like useMouseEvents: false, could work for this.

**Comment by rolexta** [comment]
can we just disable small flipping effect when cursor hovering on each corner? 
...and swipe event still work
